spec/support/factory_girl.rb

```diff
@@ -35,4 +35,17 @@
     bar
     user
   end
+
+  factory :vote do
```

Comment:
nice factories. You might consider splitting out separate active record objects into separate factory files.